generate.sh

```diff
@@ -1,84 +1,14 @@
 # #!/bin/bash
 set -euo pipefail
 
-output_file="passage.gen.go"
+if [ -z "$1" ]; then
+  echo "Required generator file is missing."
+  exit 1
+fi
 
-# Skip pointer on optional fields
-fields="\
```

Comment:
all of these are covered by https://github.com/passageidentity/passage-go/blob/777d8958f93e4f4c7404979fa7806b6d3618bbc9/overlay.yml#L87-L122